Runtime/Patterns/Scene/SceneLoader.cs

```diff
@@ -14,7 +13,6 @@ public static class SceneLoader
         public static event Action<Scene, LoadSceneMode> SceneLoaded = delegate { };
 
         static readonly List<Scene> s_AdditiveScenes = new List<Scene>();
-        static readonly Dictionary<string, AsyncOperation> s_LoadSceneOperations = new Dictionary<string, AsyncOperation>();
```

Comment:
Please go over all the files and make sure you haven't removed a valid features like I was saying before.